### ISSUE TITLE
Opening the Page Info via the identity panel / padlock doesn't open it in the sidebar

### ISSUE THREAD

**Issue by jobbautista9** [FranklinDM/TGS#74]
Instead it still tries to open it in a new window.

**Comment by fofajardo** [comment]
Yeah, the extension doesn't override the identity handler's behavior (yet).

This involves replacing the code run by the `gIdentityHandler.handleMoreInfoClick` function and should remain configurable in the extension's preferences when implemented.
